geminimd/ui.py

```
@@ -0,0 +1,133 @@
+                self.console.print(result.stdout.rstrip())
+                return
+        except Exception:
+            pass
+        self.console.print(Panel(text, title=title))
+
+    # Prompting ------------------------------------------------------------
+    def confirm(self, prompt: str, *, default: bool = True) -> bool:
+        if self.plain:
+            return default
+        try:
+            result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)